pkg/apis/eksctl.io/v1alpha4/validation.go

```diff
@@ -132,5 +133,33 @@ func ValidateNodeGroup(i int, ng *NodeGroup) error {
 		return err
 	}
 
+	if err := validateNGSSHFlags(ng.SSH); err != nil {
+		return errors.New("only one ssh public key can be specified per node-group")
+	}
+	return nil
+}
+
+func validateNGSSHFlags(SSH *NodeGroupSSH) error {
```

Comment:
this name looks a bit weird, any ideas for a better name?

Reply:
Suggestion 1: Omit `Flags` because it is validating the NodeGroupSSH (right now only flags, sure).
Suggestion 2: Either spell out NodeGroup like `validateNodeGroupIAM` or all the private functions of ValidateNodeGroup could omit the implied 'NodeGroup', and just be `validateSSH`?
NOTE: Feel free to ignore both.

Reply:
Thanks!